lms/djangoapps/instructor/enrollment.py

```diff
@@ -267,6 +269,15 @@ def reset_student_attempts(course_id, student, module_state_key, requesting_user
 
     if delete_module:
         module_to_reset.delete()
+        if block and block.has_score:
```

Comment:
My guess is that it's fine to fire SCORE_CHANGED for blocks not in the modulestore, and to assume a `max_score` of 0 because it is no longer possible to score any points for the block?